core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -2182,6 +2183,9 @@ public void testReplaceTransactionRequiresTableExists() {
   public void testConcurrentReplaceTransactions() {
     C catalog = catalog();
 
+    // TODO: temporarily ignore this test for REST catalogs (issue #8390)
+    Assumptions.assumeFalse(catalog instanceof RESTCatalog);
```

Comment:
@aokolnychyi, while I understand wanting to get this in, I don't think it's a good idea to set a precedent that we will disable tests that aren't passing. I also don't see much description of the problem on #8390.